.github/workflows/ci.yml

```diff
@@ -27,3 +27,8 @@ jobs:
       run: |
         tox
 
+    - name: Check formatting
+      run: |
+        python -m pip install yapf
```

Comment:
Does this mean the presubmit check on github will fail if one didn't format the code?

Reply:
Oh I see your comment, great!